recipes-bsp/patch-u-boot-env/files/patch-u-boot-env.config

```diff
@@ -0,0 +1,9 @@
+ustate=0
+sysselect=1
+toggle_active=echo 'Switching active partition ...'; if test $sysselect = 1;then setenv sysselect 2; else setenv sysselect 1; fi; echo Active system is now $sysselect;
+
+update_sysselect=if env exists ustate && env exists sysselect; then if test $ustate = 0; then echo SWUpdate: no update pending.; elif test $ustate = 1; then echo SWUpdate: ustate = TEST ; setenv ustate 2; run toggle_active; saveenv; elif test $ustate = 2; then echo SWUpdate: ustate = FALLBACK; setenv ustate 3; run toggle_active; saveenv; elif test $ustate = 3; then echo SWUpdate: still in FALLBACK; else echo SWUpdate: invalid ustate; fi; setenv distro_bootpart ${sysselect}; if fstype ${devtype} ${devnum}:${distro_bootpart} bootfstype; then run scan_dev_for_boot; fi; else echo SWUpdate: some variables are missing!; fi
```

Comment:
Is it by intention that the states are named diverging from SWUpdate's? 

For example `ustate==1` is `INSTALLED` not `TEST` as printed. Same goes for `ustate==3` which is `FAILED` not `FALLBACK`.